bin/pulumilocal

```diff
@@ -20,22 +21,13 @@ if os.path.isdir(os.path.join(PARENT_FOLDER, '.venv')):
 # define global constants
 TRUE_STRINGS = ['1', 'true', 'True']
 PULUMI_CMD = os.environ.get('PULUMI_CMD') or 'pulumi'
-PULUMI_STACK_NAME = os.environ.get('PULUMI_STACK_NAME') or 'localstack'
 LOCALSTACK_HOSTNAME = os.environ.get('LOCALSTACK_HOSTNAME') or 'localhost'
 EDGE_PORT = int(os.environ.get('EDGE_PORT') or '4566')
 USE_SSL = os.environ.get('USE_SSL') in TRUE_STRINGS
 
```

Comment:
I was using PULUMI_CMD in my `ecs-multi-stack` project. This caused a loop where pulumilocal was calling pulumilocal.